### ISSUE TITLE
Trigger Azure DevOps Pipeline now working with MacOS 

### ISSUE THREAD

**Issue by JasWoolieX** [maikvandergaag/msft-extensions#264]
I am getting below issue when I integrated this plugin with our build tasks.
#[error]The current operating system is not capable of running this task. That typically means the task was written for Windows only. For example, written for Windows Desktop PowerShell.

![image](https://user-images.githubusercontent.com/42925456/109094098-903d2900-776d-11eb-94cd-632cb75268b1.png)

Is this a known issue at your end?

**Comment by maikvandergaag** [comment]
Yes this is a known issue. I have it on the backlog to rewrite it to also support other operating systems.
